### DISCUSSION TITLE
SSH connection between Windows 10 Vs Code and code-server

### DISCUSSION THREAD

**Discussion by tjatkowski** [coder/code-server#4449] [Q&A]
I am using standard Visual Studio Code on my Windows 10 pc. I was wondering if I can use code-server on my android tablet and connect with my visual studio code on PC and control it remotely.
Is SSH connection between Original vscode and code-server supported?

**Comment by lg-nineoneninethreenine** [comment]
If you use a bridge like ngrok or cloudflare, and bridge your SSH port, it might allow you to SSH into your instance.

**Reply by jsjoeio** [reply]
^you could try this!

If you're referring to the official Remote SSH extension in VS Code, I'm not sure if it works with code-server 🤔 You could try and report back
